lib/Pipes/PipelineManager.cpp

```diff
@@ -142,6 +143,9 @@ PipelineManager::create(llvm::ArrayRef<std::string> Pipelines,
     return MaybePipeline.takeError();
 
   Manager.recalculateAllPossibleTargets();
+  if (auto Error = Manager.computeDescriptor(); Error)
```

Comment:
1. Add a blank line, computing the descriptor is not related to do with `recalculateAllPossibleTargets`.
2. There's some code duplication going on here, can we have a `PipelineManager::finalizeIntialization` `private` method that does `recalculateAllPossibleTargets` and `computeDescriptor` (and maybe other stuff)?

Reply:
1. Ok
2. Moved some code around, there's no more duplication